pm2.config.js

```diff
@@ -2,8 +2,11 @@ module.exports = {
   apps: [
     {
       name: 'tug-website-next-server',
+      // call next CLI directly otherwise PM2 cluster won't work
       script: 'node_modules/.bin/next',
       args: 'start',
+      // should be the current folder, otherwise PM2 will readlink first and break reload
+      cwd: process.env.PWD,
```

Comment:
What's the meaning of `readline`? Did you spell it wrong? Also, I am still confusing about this change because even without this config, reload seems to be working fine on my local.

Reply:
No it’s readlink, a command in Linux.